### PR TITLE
Update spell checker and fix some typos

### PR DESCRIPTION
**Pull request by yufeih**
https://github.com/check-spelling/check-spelling/issues/14

### CODE REVIEW

**Comment by jsoref** [comment]
Fwiw, I do intend to fix the flagging of `we'd`, I think it's fixed in prerelease, but that's still at least a week away from a release... I'm actively working on some pieces...

File: docs/specs/validation/table-validation.yml [hunk 1]
```diff
@@ -124,13 +124,13 @@ inputs:
     |------------|-----------------------------|---------------------------------|  
     |Move|ALT|ALT|
   b.md: |
-    this is an incluede title
+    this is an include title
```
Comment (jsoref) [inline docs/specs/validation/table-validation.yml:127]:
```suggestion
    this is an included title
```

File: docs/specs/validation/table-validation.yml [hunk 2]
```diff
@@ -124,13 +124,13 @@ inputs:
...
   c.md: |
     ||TEXT|C|  
     |------------|-----------------------------|---------------------------------|  
     |[!INCLUDE[include1](b.md)]|ALT|ALT|
   d.md: |
-    **this is an incluede title**
+    **this is an include title**
```
Comment (jsoref) [inline docs/specs/validation/table-validation.yml:133]:
```suggestion
    **this is an included title**
```

File: docs/specs/schema.yml [hunk 3]
```diff
@@ -138,7 +138,7 @@ outputs:
   .errors.log: |
     {"message_severity":"error","code":"unexpected-type","message":"Expected type 'String' but got 'Object'.","file":"docs/a.yml","line":3,"column":18}
 ---
-# Schema documents with Href content referencing resource file, the resource file should be outputed
+# Schema documents with Href content referencing resource file, the resource file should have output
```
Comment (jsoref) [inline docs/specs/schema.yml:141]: I'm not sure about this. Possibly:
```suggestion
# Schema documents with Href content referencing resource file, the output should be the resource file
```

File: docs/specs/validation/heading-validation.yml [hunk 4]
```diff
@@ -87,7 +87,7 @@ inputs:
       }
     }
   a.md: |
-    # Title 1 With Hashe
+    # Title 1 With Hash
     ## Title 2 With Hashes
```
Comment (jsoref) [inline docs/specs/validation/heading-validation.yml:91]: Either
```suggestion
    # Title 1 With Hashes
    ## Title 2 With Hashes
```

Comment (jsoref) [reply docs/specs/validation/heading-validation.yml:91]: Or:
```suggestion
    # Title 1 With Hash
    ## Title 2 With Hash
```

**Review by jsoref — COMMENTED** [review]
Some thoughts

**Review by mingwli — APPROVED** [review]
😲

**Pull request merged**